File: patch_phase6.py
import re

with open('data/prepare-embeddings.ts', 'r') as f:
    content = f.read()

old_text = 'if (!shouldSkip(\"Pre-computed embeddings\", EMBEDDINGS_BIN)) {'
new_text = 'if (false) { // Embeddings skipped'

content = content.replace(old_text, new_text)

with open('data/prepare-embeddings.ts', 'w') as f:
    f.write(content)

print('Patched')

File: patch-phase6.js
const fs = require('fs');
let c = fs.readFileSync('data/prepare-embeddings.ts', 'utf8');
c = c.replace('if (!shouldSkip("Pre-computed embeddings", EMBEDDINGS_BIN)) {', 'if (false) { // Embeddings skipped');
fs.writeFileSync('data/prepare-embeddings.ts', c);
console.log('Skipped Phase 6');
